README.md

```diff
@@ -1 +1,28 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# About Me
+
+## Bio
+
+![*This is me*](Picture_hw1.jpg)
+
+### *Introduction*
+
+My name is Francesca, and I am a rising fourth year student who is majoring in Economics and minoring in Health and Society. I am currently located in Missouri. I am taking this class to learn how to use R and to be able to use it for future classes, research, and projects.
+
+### *Hobbies/Interests*
+
+*   **In my free time I enjoy...**
+
+    * *Cooking*
+    * *Playing Soccer*
+    * *Doing Puzzles*
+    * *Gardening*
+
+*   **A song I like is...**
+
+    * [Super Trouper](https://www.youtube.com/watch?v=QTmEFtGeYqQ)
+    
+*   **A show I've watched recently is...**
+    
+    * [Survivor](https://www.imdb.com/title/tt0239195/)
```

Comment:
Fun Fact: Adam Klein from Millennials vs Gen X and other seasons went to my middle and high school.